composer.json

```
@@ -42,7 +59,9 @@
         "drush/drush": "^10.3"
     },
     "require-dev": {
-        "metadrop/drupal-dev": "^0.1.10"
+        "metadrop/drupal-dev": "^0.1.10",
+        "metadrop/grumphp-php-compatibility": "master",
+        "phpcompatibility/php-compatibility": "dev-develop as 9.3.6"
```

Tag 9.3.6 does not exist; https://github.com/PHPCompatibility/PHPCompatibility/tags
```suggestion
        "phpcompatibility/php-compatibility": "dev-develop as 9.3.5"
```